### DISCUSSION TITLE
Request for input interface suitable for databases with inconstant atom number.

### DISCUSSION THREAD

**Discussion by permissionx** [deepmodeling/deepmd-kit#300] [Q&A] [answered]
Seems present interface doesn't  allow the atom number change in the input .raw files. Is it possible to modify the input rule for the situations that atom number varies between frames?

**Comment by njzjz** [comment, marked as answer]
This rule is for performance. You can divided your data into different directory (i.e. different systems). `systems` parameter can be a list.
An example here: https://doi.org/10.6084/m9.figshare.12973055

**Comment by permissionx** [comment]
Got it, thanks.
